megalista_dataflow/sources/json_execution_source.py

```diff
@@ -48,8 +48,12 @@ def read(self, range_tracker):
     mcc = False if mcc_json is None else mcc_json
     app_id = self._json_config.get_value(json_data, "AppId")
     google_analytics_account_id = self._json_config.get_value(json_data, "GoogleAnalyticsAccountId")
-    campaign_manager_account_id = self._json_config.get_value(json_data, "CampaignManagerAccountId")
-    account_config = AccountConfig(google_ads_id, mcc, google_analytics_account_id, campaign_manager_account_id, app_id)
+    campaign_manager_profile_id = self._json_config.get_value(json_data, "CampaignManagerProfileId")
+    
+    if campaign_manager_profile_id is None:
+    campaign_manager_profile_id = self._json_config.get_value(sheet_id, "CampaignManagerAccountId")
```

Comment:
considering that it is inside an if clause, an identation would be needed here

Reply:
Done!